src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -137,28 +133,242 @@ public static String getProgressBar(Track track) {
+                message.getCreationTime().toInstant().toEpochMilli()
+        ));
+    }
+
+    /**
+     * Gets the String representing the verification level.
+     *
+     * @param level The verification level to get the string version.
+     * @return A string representing the verification level.
+     */
+    public static String getVerificationString(Guild.VerificationLevel level) {
+        switch (level) {
+            case HIGH:
+                return "(\u256F\u00B0\u25A1\u00B0\uFF09\u256F\uFE35 \u253B\u2501\u253B"; //(╯°□°）╯︵ ┻━┻
+            case VERY_HIGH:
+                return "\u253B\u2501\u253B\u5F61 \u30FD(\u0CA0\u76CA\u0CA0)\u30CE\u5F61\u253B\u2501\u253B"; //┻━┻彡 ヽ(ಠ益ಠ)ノ彡┻━┻
+            default:
+                return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
+        }
+    }
+
+    /**
+     * Parses time from string.
+     *
+     * @param time The time string to parse.
+     * @return The long representing the time entered.
+     */
+    public static Long parseTime(String time) {
+        Matcher digitMatcher = timeRegex.matcher(time);
+        if (digitMatcher.matches()) {
+            try {
+                return new PeriodFormatterBuilder()
+                        .appendHours().appendSuffix(":")
+                        .appendMinutes().appendSuffix(":")
+                        .appendSeconds()
+                        .toFormatter()
+                        .parsePeriod(time)
+                        .toStandardDuration().getMillis();
+            } catch (IllegalArgumentException e) {
+                return null;
+            }
+        }
+        PeriodFormatter formatter = new PeriodFormatterBuilder()
+                .appendHours().appendSuffix("h")
+                .appendMinutes().appendSuffix("m")
+                .appendSeconds().appendSuffix("s")
+                .toFormatter();
+        Period period;
+        try {
+            period = formatter.parsePeriod(time);
+        } catch (IllegalArgumentException e) {
+            return null;
+        }
+        return period.toStandardDuration().getMillis();
+    }
+
+    /**
+     * Coverts the json from redis to a RedisMessage.
+     *
+     * @param json The son to convert
+     * @return {@link RedisMessage}
+     */
+    public static RedisMessage toRedisMessage(String json) {
+        Pair<List<String>, List<String>> paths = jsonContains(json,
+                "messageID",
+                "authorID",
+                "channelID",
+                "guildID",
+                "content",
+                "timestamp"
+        );
+        if (paths.getKey().size() != 6) {
+            throw new IllegalArgumentException("Malformed JSON! Missing paths: " +
+                    Arrays.toString(paths.getValue().toArray(new String[paths.getValue().size()])));
+        }
+        return FlareBot.GSON.fromJson(json, RedisMessage.class);
+    }
+
+    /**
+     * Converts an embed into a String
```

Comment:
How does it do it? What does it handle? How does it handle things like the title? Fields? Does the footer do anything?